### ISSUE TITLE
Many colored light sources will assume white color

### ISSUE THREAD

**Issue by mugg91** [transhumandesign/kag-base#1950]
##Description

1) Apply https://github.com/transhumandesign/kag-base/pull/1916
2) Place lots of lamps on the same places and turn them on.
3) Notice they are white instead of being team colored.

## Details

![CCC](https://github.com/transhumandesign/kag-base/assets/14877281/dd5e3d11-7ac9-4b72-82dc-e1dc4d052d9a)

In the above screenshot, all lamps are turned on.
On the left side they are more white-colored.
I expected a result resembling the following screenshot:

![DDD](https://github.com/transhumandesign/kag-base/assets/14877281/b7e3f151-48dc-4bd2-9255-3592530b8b02)

In short, I'd like the light sources not to add their color values until they reach 0xffffff.
They should just stack on top of each other while keeping their color value.
